### PR TITLE
Add guide for use with QwikCity projects

### PR DESCRIPTION
**Pull request by riceboyler**
I recently started playing with Qwik and wanted to create a simple guide for using with QwikCity projects. FWIW, I have absolutely nothing to do with the development of Qwik.

### CODE REVIEW

File: website/docs/guides/qwik.mdx
Comment (jwr1) [inline website/docs/guides/qwik.mdx]: Package installation is not needed here as that's already covered in "Getting Started".

```suggestion
In the `routes/layout.tsx` file, simply import those weights you need (or if variable, you can import the variable).
```

File: website/docs/guides/qwik.mdx
Comment (jwr1) [inline website/docs/guides/qwik.mdx:1]: Also make sure to add an entry to the [docsList.json](https://github.com/fontsource/fontsource/blob/main/website/src/configs/docsList.json) file, alphabetically sorted. Something like the following:

```json
  {
    "key": "qwik",
    "title": "Qwik",
    "path": "/docs/guides/qwik"
  },
```

**Review by jwr1 — COMMENTED** [review]
Thanks for the PR :tada:. We are currently in the process of rewriting the website, so we'll have to copy this over again later on, but it certainly helps out as I've never used Qwik before.

**Comment by ayuhito** [comment]
I think we can keep this PR in stasis until #597 is merged. The docs are going for a full rewrite and we're adding a new guides section specially for all sorts of integrations like this.

Thanks for the contribution though, I've been following Qwik's work for a while!

File: website/docs/guides/qwik.mdx
Comment (riceboyler) [reply website/docs/guides/qwik.mdx:1]: I'll hold off on updates to this PR until #597 is merged. The beta site looks great!

FWIW, I just started playing with Qwik the last few weeks and it's really impressive that it can solve the problems it's trying too. Still very early, but there's lots of promise there. 

Thank you all for making such an incredibly helpful tool in Fontsource! I hope to be able to make small contributions to say thanks.

**Comment by ayuhito** [comment]
Hi @riceboyler, thanks for the PR! The docs are largely ready for the new website so it would be awesome if you could recreate this PR with the `website` branch as the target. I think we can sneak this in with the official release.

I would also suggest switching the default imported font in your guide with the variable variants since we want to promote the usage of variable fonts over regular fonts. For any other issues, we can handle them in a separate review. 👍

File: website/docs/guides/qwik.mdx
Comment (ayuhito) [inline website/docs/guides/qwik.mdx:33]:
```suggestion
---
title: Qwik
contents: Frameworks
---

## Qwik

---

To use Fontsource with a [QwikCity](https://qwik.builder.io) project, it is suggested to use the `/routes/layout.tsx` file.

### Import

Once you install the Fontsource package of your choice, you can import the chosen font variant into `routes/layout.tsx`:

```tsx
import "@fontsource-variable/open-sans/wght.css";

export default component$(() => {
  return (
    <>
      <Header />
      <main>
        <Slot />
      </main>
      <Footer />
    </>
  );
});
```
```

**Review by ayuhito — CHANGES_REQUESTED** [review]
I went ahead and updated this branch with `main` now that the website is live. All you need to do is add Qwik to [`docs/sidebar.json`](https://github.com/fontsource/fontsource/blob/main/website/docs/sidebar.json) as well as accept the below suggestions and this PR can be merged!

**Comment by ayuhito** [comment]
Closed in favour of #744. Thanks for the PR!

**Pull request closed without merging**
